src/index.ts

```diff
@@ -1 +1,51 @@
-console.log('test');
+import Home from './pages/Home';
+import Detail from './pages/Detail';
+import NotFound from './pages/NotFound';
+import Slash from './pages/Slash';
+import Simplicity from './pages/Simplicity';
+
+export const navigate = (url: string) => {
+	window.history.pushState(null, null, url);
+	render();
+};
+
+export const createElement = (domString: string) => {
+	const $temp = document.createElement('template');
+	$temp.innerHTML = domString;
+	return $temp.content;
```

Comment:
다음과 같이 메모리 해제를 해 주어 메모리 누수를 방지하면 더 좋을 것 같습니다.

```suggestion
    const content = $temp.content;
    $temp.remove();  // 메모리 해제를 위해 요소 제거
    return content;
```

Reply:
@f-lab-joshua 
넵 감사합니다!